locust/test/test_web.py

```diff
@@ -257,3 +258,37 @@ def my_task(self):
         )
         self.assertEqual(200, response.status_code)
         self.assertIn("Step Load Mode", response.text)
+
+
+class TestWebUIAuth(LocustTestCase):
+    def setUp(self):
+        super(TestWebUIAuth, self).setUp()
+
+        parser = get_parser(default_config_files=[])
+        self.environment.options = parser.parse_args([])
+        self.runner = LocustRunner(self.environment, [])
+        self.stats = self.runner.stats
+
+        self.web_ui = WebUI(self.environment)
+        self.web_ui.app.view_functions["request_stats"].clear_cache()
+        self.setUpBasicAuth()
+        gevent.spawn(lambda: self.web_ui.start("127.0.0.1", 0))
+        gevent.sleep(0.01)
+        self.web_port = self.web_ui.server.server_port
+
+    def setUpBasicAuth(self):
```

Comment:
If we move the logic of setting up basic auth to web.py as suggested in [this comment](https://github.com/locustio/locust/pull/1313/files#r403993789) we don't need a special method for setting up the BasicAuth.

Reply:
You're right, refactored and removed this method now.